method-discover.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * docs/getting-started-guide/master.adoc
+
+:_content-type: CONCEPT
+[id="method-discover_{context}"]
+= Discover phase
+
+.Red Hat migration methodology: Discover phase
+image::RHAMT_AMM_Methodology_446947_0617_ECE_Discover.png[Red Hat Migration Methodology: Discover]
+
+The _Discover_ phase is when you gather all stakeholders and decision-makers together to understand the current state and business drivers, and to determine the migration or modernization needs.
+
+In this phase, you explore technologies and discuss potential approaches. Identify the existing pain points, concerns, requirements, and some potential challenges. Define the high-level business priorities and scope of the assessment. Determine in what ways you want to modernize your application development and delivery to allow you to innovate more rapidly.
```

Comment:
@HagayVider1 
I think this should be a bulleted list:
"In this phase, you explore technologies and discuss potential approaches:
* Identify the existing pain points, concerns, requirements, and potential challenges 
* Determine the high-level business priorities and scope of the assessment
* Determine in what ways you want to modernize your application development and delivery to enable faster innovation"